proxy/proxy.go

```diff
@@ -21,6 +21,7 @@ import (
 	"time"
 	"unicode/utf8"
 
+	"golang.org/x/exp/maps"
```

Comment:
This is not in stdlib, see https://pkg.go.dev/maps

Reply:
Yes, but there is no `maps.Keys` method on the stdlib page.
![image](https://github.com/zalando/skipper/assets/8851784/72f0141b-d066-4ac3-9277-72472cae3030)
https://pkg.go.dev/maps

The experimental for comparison.
![image](https://github.com/zalando/skipper/assets/8851784/748e9fc5-f280-46eb-9a19-407b24dd3dcc)
https://pkg.go.dev/golang.org/x/exp/maps

Reply:
Right, indeed https://github.com/golang/go/issues/61538. Thanks for checking